codespell_lib/data/dictionary.txt

```diff
@@ -16979,7 +16985,7 @@ messges->messages
 messsage->message
 messsages->messages
 messure->measure
-messured->measurd
```

Comment:
Can you add `measurd` as a typo, probably suggested -ed and -e

Reply:
added